locust/contrib/fasthttp.py

```diff
@@ -438,7 +438,7 @@ class FastResponse(CompatResponse):
 
     _response: HTTPSocketPoolResponse | None = None
 
-    encoding: str | None = None
+    encoding: str | float | None = None
```

Comment:
When is encoding a float?

Reply:
This is just one the type checks were balking at:

Technically using `chardet` 's `detect` method as a fallback to try and find out the encoding, the returned dictionary _values_ for the key _encoding_ (`Dict[str, str | float | None]`) can be `str | float | None` so it was complaining 🙃  

<img width="1137" alt="image" src="https://github.com/locustio/locust/assets/82032454/2eddc0ab-9d60-43e5-8ffd-383a55cdaec0">

Reply:
It is probably super unlikely it's just technically a possibility at least in terms of permissable types